src/libraries/System.Data.Odbc/src/System/Data/Odbc/ODBC32.Common.cs

```diff
@@ -3,12 +3,15 @@
 
 namespace System.Data.Odbc
 {
+    // Class needs to be public to support serialization with type forwarding from Desktop to Core.
     [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public static class ODBC32
+    public static partial class ODBC32
     {
+        // from .\public\sdk\inc\sqlext.h: and .\public\sdk\inc\sql.h
+        // must be public because it is serialized by OdbcException
         [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
         public enum RETCODE : int
-        {
+        { // must be int instead of short for Everett OdbcException Serializablity.
```

Comment:
This comment can be removed.  Changing the underlying type of any public enum is a breaking change.

Reply:
@stephentoub its already ```int``` in the master. we are just moving around the type to avoid duplicate code.
I will remove the comment

Reply:
Yup, understood it's already int; I was just clarifying why the comment is superfluous.

Reply:
ah okay. sry about that :)